gw_spaceheat/requirements/base.in

```diff
@@ -2,5 +2,5 @@ pendulum
 pika==1.1.0
 paho-mqtt
 pytz
-
+python-dotenv
```

Comment:
Let's merge this PR, but also update the requirements/base.txt using pip-tools:

- python -m pip install pip-tools
- from gw_spaceheat dir: pip-compile --output-file=requirements/base.txt requirements/base.in

Reply:
1. Here is the resulting change from running the `pip-compile` command: https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/3/commits/6812183c5281017e8fdf97961da723d029c390e5
2. I went ahead and added the helper function we discussed to utils (I can delete this if you don't want it): https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/3/commits/7e650895992faebdf31fe3aa0fa8b09b8b7995ce
So you would call it like this:
```
from utils import get_secret_value

## if MQTT_PW is in your .env, then the value will just print out
print(get_secret_value('MQTT_PW'))

## if NONEXISTENT_PW is not in your .env file, and exception will be raised
print(get_secret_value('NONEXISTENT_PW'))
```

3.  I deleted the example file so we don't have clutter in the codebase: https://github.com/thegridelectric/gw-scada-spaceheat-python/pull/3/commits/2634ee6ce3188d103f47a201889d0b582fc69f6a